src/index.js

```diff
@@ -160,9 +169,12 @@ class Masq {
     })
   }
 
-  async _isRegistered () {
-    // is registered if the userId is the id of a known db
-    return !this.userId || utils.dbExists(this.userId)
+  async _isRegistered (userId) {
+    // is registered if :
+    // - this.userId is set (loaded in Masq constructor)
+    // - this.userId is the same as userId
+    // - there is a db for this userId
+    return (this.userId && this.userId === userId && utils.dbExists(userId))
```

Comment:
In this case, why take a param named `userId` ? the result of this function does not depend on it it seems. It only depends on `this.userId`

Reply:
We do check that the input userId is equal to attrubute this.userId

Reply:
We could only check if dbExists(userId)